evm/lightchain_shell.py

```diff
@@ -28,6 +29,7 @@
 parser = argparse.ArgumentParser()
 parser.add_argument('-db', type=str, required=True)
 parser.add_argument('-debug', action='store_true')
+parser.add_argument('-default_shell', action='store_true', default=False)
```

Comment:
It was only after I read the implementation that I understood what default means here. Maybe vanilla_shell would be a better name for it?

Reply:
Rename done.